docker/scripts/utils.ts

```diff
@@ -1,14 +1,28 @@
 import { execSync, ExecSyncOptions } from 'child_process';
 import { statSync, readFileSync, accessSync, constants } from 'fs';
-import omit from 'lodash/omit';
+import has from 'lodash/has';
 import { logger } from '@api3/airnode-utilities';
 import { goSync } from '@api3/promise-utils';
 
+const hideSecrets = (command: string, options?: ExecSyncOptions) => {
+  // There are currently 2 secrets being logged:
+  // * DockerHub auth token - stdin
+  // * GitHub auth token - remote URL
+  // Third secret, the NPM auth token, is stored in a file therefor not logged
+
+  // DockerHub auth token
+  const hiddenOptions = has(options, 'input') ? { ...options, input: '***' } : options;
+
+  // GitHub auth token
+  const hiddenCommand = command.replace(/(https:\/\/x-access-token):(.*?)(@github\.com.*$)/, '$1:***$3');
```

Comment:
Btw. is the token really logged? I remember when I was doing some CI stuff on my personal projects that GitHub was able to star ("****") the secrets. Even if they were output as part of the message. Doesn't that happen here?